filebeat/fileset/config.go

```diff
@@ -0,0 +1,17 @@
+package fileset
+
+// ModuleConfig contains the configuration file options for a module
+type ModuleConfig struct {
+	Module   string                    `config:"module"     validate:"required"`
+	Enabled  *bool                     `config:"enabled"`
+	Filesets map[string]*FilesetConfig `config:"filesets"`
```

Comment:
Not sure I understand the question. Whether we need `Filesets` part of the `ModuleConfig`?